src/WebJobs.Script.WebHost/Management/IFunctionsSyncManager.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Management
+{
+    public interface IFunctionsSyncManager
+    {
+        /// <summary>
+        /// Sync function triggers with Antares infrastructure.
+        /// </summary>
+        /// <param name="checkHash">Indicates whether the last sync hash should be checked
+        /// to conditonally perform the sync.</param>
+        /// <returns>Success result and error info.</returns>
+        Task<(bool success, string error)> TrySyncTriggersAsync(bool checkHash = false);
```

Comment:
nit: With a new interface, would recommend avoiding the tuple here and have a type with these properties (e.g. `SyncTriggersResult` or something along those lines)